src/components/staking/components/BalanceBreakdown.js

```diff
@@ -123,7 +124,7 @@ function BalanceBreakdown({ total, onClickAvailable, availableType, error }) {
                         </div>
                     </div>
                     <div className='title'>
-                        <div onClick={() => setOpen(!open)}>
+                        <div onClick={() => {setOpen(!open); Mixpanel.track("Watch available to send")}}>
```

Comment:
This will track on open and close right now

Reply:
yes, so I do not track open or close. But track checkout available token

Reply:
Right, maybe adjust the language otherwise it seems that it's only tracking when the user expands the dropdown. `STAKE Click toggle available to stake/unstake dropdown`